packages/core-connector-interface/package.json

```diff
@@ -15,27 +16,13 @@
     "node": "12",
     "yarn": "1"
   },
-  "scripts": {
-    "lint": "prettier --check ."
-  },
   "types": "src/index.d.ts",
-  "prettier": {
-    "printWidth": 160,
-    "semi": false,
-    "singleQuote": true,
-    "quoteProps": "consistent",
-    "arrowParens": "avoid"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "yarn lint"
-    }
+  "scripts": {
+    "build": "echo 'Nothing to build'"
```

Comment:
Why do we need placeholder scripts?

Reply:
prettier output, without it you get errors in red, although I will try to not use `echo`

Reply:
what causes errors?

Reply:
if a script "build" does not exist, it will fail with error, I want to make sure that if something fails it's an unexpected error and not something that is not truly an error